src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/QueryIT.java

```diff
@@ -99,6 +99,49 @@ public void multipleFromTest() throws IOException {
         Assert.assertEquals(14, getTotalHits(response));
     }
 
+    @Test
+    public void selectAllAndSpecificFieldTest() throws IOException {
+        String[] arr = new String[] {"account_number", "firstname", "address", "birthdate", "gender", "city", "lastname", "balance", "employer", "state", "age", "email", "male"};
+        Set<String> expectedSource = new HashSet<>(Arrays.asList(arr));
+        JSONObject response = executeQuery(String.format(Locale.ROOT,
+                "SELECT *, age FROM %s/account", TestsConstants.TEST_INDEX_BANK));
+        Assert.assertTrue(response.has("hits"));
+        JSONArray hits = getHits(response);
+        for (int i = 0; i < hits.length(); ++i) {
+            JSONObject hit = hits.getJSONObject(i);
+            Assert.assertEquals(expectedSource, getSource(hit).keySet());
```

Comment:
Do you want to also verify the absence of a duplicate field `age` in the result set? Since you're using sets, you only make sure there's at least one.

Reply:
The test case accounts for that- if there is a duplicate field, the `assertEquals()` statement will fail against `arr `

Reply:
The code compares two sets, which never have duplicates.

Reply:
I think we should give alias to column `age` in the SELECT.